<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FriendList</class>
 <widget class="QWidget" name="FriendList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>871</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255,0%)</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>411</width>
     <height>871</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(147,181,207);
border-radius: 10px;</string>
   </property>
   <widget class="QLabel" name="label_top">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_min">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
   color:rgb(134, 134, 134);
	font: 12pt &quot;Adobe 黑体 Std R&quot;;
}
QLabel:hover{
   background-color:rgb(58,149,222);
   color:white;

}</string>
    </property>
    <property name="text">
     <string>一</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_close">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
   color:rgb(134, 134, 134);
	font: 12pt &quot;Adobe 黑体 Std R&quot;;
}
QLabel:hover{
  background-color:rgb(212, 64, 39);
  color:rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>391</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  rgb(226,225,228);
QLineEdit{

background:rgb(226,225,228);
border: 1px solid rgb(134, 134, 134);
border-radius:2px;
	color: rgb(134, 134, 134)
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>371</width>
       <height>191</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>11</pointsize>
          <weight>3</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
font: 25 11pt &quot;微软雅黑 Light&quot;;
  background-color:rgb(0,0,0,0%);
color:rgb(134, 134, 134);
}
QLabel:hover{
  background-color:rgb(0,0,0,0%);
}</string>
        </property>
        <property name="text">
         <string>               用户名</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_4">
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{

background:rgb(226,225,228);
border: 1px solid rgb(134, 134, 134);
border-radius:2px;
	color: rgb(134, 134, 134)
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>11</pointsize>
          <weight>3</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
font: 25 11pt &quot;微软雅黑 Light&quot;;
  background-color:rgb(0,0,0,0%);
color:rgbrgbrgb(134, 134, 134);
}
QLabel:hover{
  background-color:rgb(0,0,0,0%);
}</string>
        </property>
        <property name="text">
         <string>     上次登录时间</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit">
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{

background:rgb(226,225,228);
border: 1px solid rgb(134, 134, 134);
border-radius:2px;
	color: rgb(134, 134, 134)
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_11">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>11</pointsize>
          <weight>3</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
font: 25 11pt &quot;微软雅黑 Light&quot;;
  background-color:rgb(0,0,0,0%);
color:rgbrgbrgb(134, 134, 134);
}
QLabel:hover{
  background-color:rgb(0,0,0,0%);
}</string>
        </property>
        <property name="text">
         <string>     当前在线时间</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{

background:rgb(226,225,228);
border: 1px solid rgb(134, 134, 134);
border-radius:2px;
	color: rgb(134, 134, 134)
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_14">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>11</pointsize>
          <weight>3</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
font: 25 11pt &quot;微软雅黑 Light&quot;;
  background-color:rgb(0,0,0,0%);
color:rgbrgbrgb(134, 134, 134);
}
QLabel:hover{
  background-color:rgb(0,0,0,0%);
}</string>
        </property>
        <property name="text">
         <string>            目标地址</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{

background:rgb(226,225,228);
border: 1px solid rgb(134, 134, 134);
border-radius:2px;
	color: rgb(134, 134, 134)
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_15">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>11</pointsize>
          <weight>3</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
font: 25 11pt &quot;微软雅黑 Light&quot;;
  background-color:rgb(0,0,0,0%);
color:rgbrgbrgb(134, 134, 134);
}
QLabel:hover{
  background-color:rgb(0,0,0,0%);
}</string>
        </property>
        <property name="text">
         <string>            目标端口</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEdit_5">
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{

background:rgb(226,225,228);
border: 1px solid rgb(134, 134, 134);
border-radius:2px;
	color: rgb(134, 134, 134)
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_16">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>11</pointsize>
          <weight>3</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
font: 25 11pt &quot;微软雅黑 Light&quot;;
  background-color:rgb(0,0,0,0%);
color:rgbrgbrgb(134, 134, 134);
}
QLabel:hover{
  background-color:rgb(0,0,0,0%);
}</string>
        </property>
        <property name="text">
         <string>                  状态</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEdit_6">
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{

background:rgb(226,225,228);
border: 1px solid rgb(134, 134, 134);
border-radius:2px;
	color: rgb(134, 134, 134)
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>391</width>
      <height>551</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(226,225,228);
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>51</y>
       <width>371</width>
       <height>461</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QListWidget
{
background: rgb(226,225,228);; border: 0px;
}
QScrollBar:vertical
{
background: rgb(226,225,228);
width:8px;
}
QScrollBar::handle:vertical
{
background: rgb(186,204,217	);
border-radius: 4px;
border: none;
min-height: 20px;
}
QScrollBar::handle:vertical:hover
{
background: rgb(128,128,128);
border: none;
min-height: 20px;
}
QScrollBar::sub-line:vertical{border: none;}
QScrollBar::add-line:vertical{border: none;}</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>11</pointsize>
          <weight>3</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
font: 25 11pt &quot;微软雅黑 Light&quot;;
  background-color:rgb(0,0,0,0%);
color:rgb(134, 134, 134);
}
QLabel:hover{
  background-color:rgb(0,0,0,0%);
}</string>
        </property>
        <property name="text">
         <string>当前在线/游戏中/离线</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_7">
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{

background:rgb(226,225,228);
border: 1px solid rgb(134, 134, 134);
border-radius:2px;
	color: rgb(134, 134, 134)
}</string>
        </property>
        <property name="text">
         <string>4/7/11</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
